crates/runtime/src/traphandlers/unix.rs

```diff
@@ -99,6 +100,42 @@ unsafe extern "C" fn trap_handler(
             return true;
         }
         info.capture_backtrace(pc);
+        // On macOS this is a bit special, unfortunately. If we were to
```

Comment:
This code is taken from https://github.com/bytecodealliance/wasmtime/pull/2723/files#diff-560b66f5eac16eef20c78bcaf2c66582dbc6cf76fa1cc72faf6bca7b736cd660L124 after renaming `Unwind` to `wasmtime_longjmp`. The comment is also adjusted accordingly.